src/drivers/memory.ts

```diff
@@ -2,12 +2,13 @@ import { defineDriver } from "./utils";
 
 const DRIVER_NAME = "memory";
 
-export default defineDriver<void>(() => {
+export default defineDriver<void, Map<string, any>>(() => {
   const data = new Map<string, any>();
 
   return {
     name: DRIVER_NAME,
-    options: {},
+    getInstance: () => data,
+    options: undefined,
```

Comment:
Was this change necessary? 

```suggestion
    options: {},
```